source/includes/steps-atlas-terraform-file-from-template.yaml

```diff
@@ -0,0 +1,242 @@
+---
+title: "In |vscode-short|, create a new file."
+ref: create-file-vscode
+level: 4
+content: |
+
+  .. list-table::
+     :header-rows: 1
+     :widths: 33 33 34
+
+     * - Operating System
+       - Method
+       - Actions
+
+     * - Any
+       - |vscode-short| Menu
+       - :guilabel:`File` > :guilabel:`New File`
+
+     * - MacOS
+       - Keyboard Shortcut
+       - Press ``Command`` + ``N``
+
+     * - Windows and Linux
+       - Keyboard Shortcut
+       - Press ``Control`` + ``N``
+
+---
+title: "Save the file to a new directory. Name the file ``main.tf``"
+ref: change-language
+level: 4
+content: |
+
+  .. note::
+
+     ``main.tf`` is the recommended filename for the entry point to a
+     Terraform module.
+
+---
+title: "In the ``main.tf`` file, type ``atlas``, then press the **Enter** or **Return** key."
+ref: bootstrap-atlas
+level: 4
+content: |
+
+  This populates the ``main.tf`` file with an example configuration 
+  using the `MongoDB Atlas
+  <https://www.terraform.io/docs/providers/mongodbatlas/index.html>`__ 
+  Terraform provider to create a Shared Tier |service| cluster. 
+
+  Update the example configuration to configure your cluster.
+
+---
+title: "Update the |service| Terraform configuration to configure your cluster."
+level: 4
+ref: update-config
+content: |
+
+  The value of the ``name`` argument of the ``mongodbatlas_project``
+  resource named ``my_project`` gains focus.
```

Comment:
I'm not really sure what you mean by "gains focus" here. I'm sure it'd be clearer if I was actively following along, but can we be a bit more descriptive here?

Is this something like "my cursor automatically focuses the value of the `name` argument in that resource's configuration file"?

Reply:
Did we define ``my_project`` somewhere earlier? Where did this resource come from?

Reply:
updated. We didn't define `my_project` -- the template provides that name for the resource. Removed this part to reduce potential confusion.